openedx/core/djangoapps/schedules/template_context.py

```diff
@@ -31,6 +31,13 @@ def encode_url(url):
 
 
 def absolute_url(site, relative_path):
+    """Add site.domain to the beginning of the given relative path.
```

Comment:
Nit: Start the sentence on the next line.   See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings